x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row.tsx

```diff
@@ -309,13 +350,19 @@ export const ExpandedRow: FC<Props> = ({ item }) => {
     );
   }
 
+  const sections: SectionConfig[] = [state, progress, stats];
+
+  if (analysisStats !== undefined) {
+    sections.push(analysisStats);
+  }
+
   const tabs = [
     {
       id: 'ml-analytics-job-details',
       name: i18n.translate('xpack.ml.dataframe.analyticsList.expandedRow.tabs.jobSettingsLabel', {
         defaultMessage: 'Job details',
       }),
-      content: <ExpandedRowDetailsPane sections={[state, progress, stats]} />,
+      content: <ExpandedRowDetailsPane sections={sections} />,
```

Comment:
I think there is too much information in the 'Job details' tab now. Maybe the content on the Stats and Analysis Stats sections should be moved out onto a new 'Stats' tab?

![image](https://user-images.githubusercontent.com/7405507/91872620-f427de80-ec6f-11ea-9bc8-80dc0dbc1a2d.png)

This would make it consistent with the Models list, which has a Stats tab in the expanded row.

The `create_time` and `model_memory_limit` seem to belong more on a 'Config' type panel than in 'Stats' but moving them out might be outside the scope of this PR.

Reply:
@peteharverson  - good call. Added the tab in 246f84299a57803af286499cab2f48c81f6be17a and updated the PR description